Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppNode.java

```diff
@@ -31,18 +33,21 @@
     public static final String SSH_INTO = "SSH into Web App (Preview)";
     public static final String PROFILE_FLIGHT_RECORDER = "Profile Flight Recorder";
 
-    private final WebApp webapp;
+    // todo: migrate file service to track2 SDK
+    private final WebApp webapp; // Track one client, keep for file service
```

Comment:
can we lazy load the webapp since the following statement will cause a network call.
```
AzureWebAppMvpModel.getInstance().getWebAppById(subscriptionId, webAppManager.id());

```